docs/_static/custom.css

```diff
@@ -67,7 +67,7 @@ code.xref.any.py {
   border-radius: 3px;
 }
 
-.rst-content dl[class]:not(.option-list):not(.field-list):not(.footnote):not(.glossary):not(.simple)>dt {
+html.writer-html5 .rst-content dl[class]:not(.option-list):not(.field-list):not(.footnote):not(.glossary):not(.simple)>dt {
```

Comment:
What is this added for?

Reply:
not sure, but without it it doesn't seem to render properly. the css is pretty insane and impossible to make sense of ... I just used the browser inspect tool to figure out what things needed to be changed to get the desired effect